### ISSUE TITLE
Wii Guitar Hero 5 Guitar isn't being mapped to Fortnite

### ISSUE THREAD

**Issue by pxdl** [InvoxiPlayGames/FestivalInstrumentMapper#84]
Trying to get a Wii Guitar Hero 5 controller working over bluetooth. Wiitarthing works fine, HidHide works fine, but when trying to play the game, no inputs are being registered. If I disable the HidHide option, it recognizes it as a regular Xbox controller, no Pro option shows up in Festival mode.

**Comment by SMHUNTERSM** [comment]
I have the same problem, were you able to solve it?

**Comment by pxdl** [comment]
@SMHUNTERSM Nope. I also bought a Raphnet adapter and that doesn't work either. Controller will not show up on Fortnite.
